TYPO3.Neos/Resources/Private/TypoScript/Prototypes/ImageUri.ts2

```diff
@@ -12,11 +14,15 @@ prototype(TYPO3.Neos:ImageUri) {
 prototype(TYPO3.Neos:ImageTag) < prototype(TYPO3.TypoScript:Tag) {
 	asset = 'pass-the-media-asset'
 	maximumWidth = 2560
```

Comment:
I think this magic number should be set to NULL in this case the image w/h will be used automatically